starknet-foundry/crates/forge/tests/integration/common/runner.rs

```diff
@@ -80,11 +80,18 @@ impl<'a> TestCase {
     const TEST_PATH: &'a str = "test_case.cairo";
     const PACKAGE_NAME: &'a str = "my_package";
 
-    pub fn from(test_code: &str, contracts: Vec<Contract>) -> Result<Self> {
+    pub fn from(test_code_or_path: &str, contracts: Vec<Contract>) -> Result<Self> {
+        let maybe_test_path = PathBuf::from(test_code_or_path);
+        let test_code = if maybe_test_path.exists() {
+            fs::read_to_string(maybe_test_path).unwrap()
+        } else {
+            test_code_or_path.to_string()
+        };
+
         let dir = TempDir::new()?;
         let test_file = dir.child(Self::TEST_PATH);
         test_file.touch()?;
-        test_file.write_str(test_code)?;
+        test_file.write_str(&test_code)?;
 
         dir.child("src/lib.cairo").touch().unwrap();
```

Comment:
I'd rather we have one consistent approach for all integration tests, otherwise it will get hard to maintain. Lets keep the previous way of having test code in the tests themselves. Otherwise all test files will just be `test_case!(path_x)` and will soon have great amount of files in data that are not obviously connected with anything.

Reply:
Sure, if you insist